web/thesauruses/ruby/3/strings.json

```diff
@@ -0,0 +1,416 @@
+{
+  "meta": {
+    "language": "ruby",
+    "language_name": "Ruby",
+    "structure": "strings",
+    "language_version": "3"
+  },
+  "concepts": {
+    "is_primitive_or_not": {
+      "name": "Is this a built-in type in this language?",
+      "code": [
+        "Yes"
+      ]
+    },
+    "import": {
+      "name": "Import the string class",
+      "code": [
+        "Built-in included in the default Modules"
+      ]
+    },
+    "default_string_byte_encoding": {
+      "name": "Default byte encoding (ex: ASCII UTF-8, UTF-16, etc.)",
+      "code": [
+        "UTF-8"
+      ]
+    },
+    "create_new_string": {
+      "name": "Create new string",
+      "code": [
+        "1. varName = \"Hello World\"",
```

Comment:
I also really want to keep "code" lines to be actual compilable code. So `varName = "Hello World"` is compilable, but the "1." at the beginning isn't. (I know you're using it to mark the line of possible options.) While maybe the system should number the lines (an issue I think I'm going to put in), for now, maybe remove the numberings and keep it to just working code.

(also the comments after should go in a "comment" block instead of "code" lines.)

Reply:
Understood! Yes, it was to make very clear the different options, I will change it and try to make it very understandable.